deployment/cdk/opensearch-service-migration/lib/service-stacks/migration-console-stack.ts

```diff
@@ -275,11 +266,37 @@ export class MigrationConsoleStack extends MigrationServiceCore {
             ...(getTargetSecretsPolicy ? [getTargetSecretsPolicy] : []),
             ...(getSourceSecretsPolicy ? [getSourceSecretsPolicy] : [])
         ]
+
         if (props.streamingSourceType === StreamingSourceType.AWS_MSK) {
             const mskAdminPolicies = this.createMSKAdminIAMPolicies(props.stage, props.defaultDeployId)
             servicePolicies = servicePolicies.concat(mskAdminPolicies)
         }
 
+        if (props.managedServiceSourceSnapshotEnabled &&
+            servicesYaml.snapshot &&
+            servicesYaml.snapshot.s3) {
+            servicesYaml.snapshot.s3.role =
+                createSnapshotOnAOSRole(this, artifactS3Arn, serviceTaskRole.roleArn,
+                    this.region, props.stage, props.defaultDeployId)
+                    .roleArn;
+            // HOW DO I GET THIS servicesYaml into the parameter AND get the service created?
```

Comment:
I'm not sure what this question is -- what do you mean by "get the service created"?